@xen-orchestra/fs/src/smb.js

```diff
@@ -1,243 +1,85 @@
-import Smb2 from '@marsaud/smb2'
```

Comment:
It must be a new handler: https://github.com/vatesfr/xen-orchestra/issues/2257#issuecomment-437297212